recipe/recipe.yaml

```
@@ -84,8 +83,8 @@ requirements:
     - pip
     - setuptools
     - pytorch
-    - ${{ "pytorch * cuda*" if cuda == "true" }}
-    - ${{ "pytorch * cpu*" if cuda == "false" }}
+    - ${{ "pytorch * cuda*" if cuda_compiler_version != "None" }}
+    - ${{ "pytorch * cpu*" if cuda_compiler_version == "None" }}
```

There's another use of the `cuda` variable further down under the run dependencies.

---

Ahh yep good catch, fixed. However it's a little unsettling that any of the builds worked...

---

As it turns out we've been silently ignoring the cuda config here. CC @wolfv